README.md

```diff
@@ -204,12 +245,12 @@ Requests/sec:  10710.55
 Transfer/sec:      1.81MB
 ```
 
-### UvReactor
+### Benchmark with UvReactor
 
 Start server:
 ```bash
 cd ./aerys
-php -n -dzend.assertions=-1 -dextension="/usr/local/opt/php70-uv/uv.so" ./vendor/bin/aerys -w 1 -c ./server.php
+php -n -dzend.assertions=-1 -dextension="/usr/local/opt/php70-uv/uv.so" ./vendor/bin/aerys -w 1 -c ./server.php  --worker-args="-n"
```

Comment:
Readme already outdated. I will remove these bash example to avoid any misunderstandings